### DISCUSSION TITLE
Wondering if anyone has successfully used The Events Calendar with FlyntWP?

### DISCUSSION THREAD

**Discussion by psquirrel2018** [flyntwp/flynt#440] [Q&A] [answered]
I installed The events Calendar into my Flynt theme and created the necessary default-template.php and events.twig files.  I followed chrismcintosh's advice here: https://github.com/timber/timber/issues/312#issue-39623334 and get my events rendered to the events.twig file.  However, most of the Events Calendar buttons don't work and those that do, take me to views that are are not complete.  For example, the next and previous buttons don't work, the date picker toggle doesn't work...  I am not getting any JS errors in the console, so I am not sure how to troubleshoot this.  Also, when I first click into the Month view, i get a partially finished view, but if I refresh, it renders more accurately, but still has some the link/button issues.  So, I am wondering if I need to also update other files such as month.php and or create more files such as month.twig...

Also, wondering if I need to load jQuery separately or something like that?

Side note, I have modified the Grid Post Latest component to pull in the tribe_events and it works perfectly.  So, I have considered just rendering all events in a grid view and adding pagination to page to next month..., but it sure would be nice to use the Calendar Month view and not have to recreate the wheel here.

Thanks,
Scott

**Comment by steffenbew** [comment]
@psquirrel2018 Unfortunately, I don't have experience working with that plugin and can't give guidance on this one.

**Comment by timohubois** [comment]
@psquirrel2018 did you find an answer regarding to your question?

**Reply by munts** [reply, marked as answer]
@timohubois,
Thank you for checking back in with me.  I was able to get it working by using the information here: https://github.com/timber/timber/issues/312#issue-39623334 as well as a few other tweaks.  The biggest issue was that Flynt was loading jQuery and the Events Calendar plugin was also loading jQuery.  Once I fixed that conflict, I was able to get the Events Calendar to render correctly.  thanks again, scott
